packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard/signup-wizard.tsx

```diff
@@ -54,18 +63,12 @@ const SignupWizard = ({ closeWizard }: TSignupWizardProps) => {
                         secondary_button_label={localize('Back')}
                         onChangeStep={onChangeStep}
                     >
-                        <Wizard.Step title='Step 1' is_fullwidth>
-                            <>
-                                <Text as='p' size='m' line-height='m' weight='bold'>
-                                    <Localize i18n_default_text='Step 1: Identity verification' />
-                                </Text>
-                                <Text as='p' size='xs' line-height='m'>
-                                    <Localize i18n_default_text="First, we'll need to verify your identity. Choose your preferred document for submission" />
-                                </Text>
-                                <Text as='p' size='xs' line-height='m'>
-                                    <Localize i18n_default_text='Note: Please ensure all your personal details are up-to-date before uploading the photo of your document.' />
-                                </Text>
-                            </>
+                        <Wizard.Step
+                            title={localize('Country of issue')}
+                            is_fullwidth
+                            is_submit_disabled={!is_country_selected}
+                        >
+                            <SelectCountryStep selectedCountry={selectedCountry} />
```

Comment:
```suggestion
                            <SelectCountryStep onSelect={onCountrySelect} />
```
@hamid-deriv It's a callback, Better to prefix the prop with `on` for better readability.